javascript/linked-list/__tests__/linked-list.test.js

```diff
@@ -295,4 +290,85 @@ test('Where both lists have different length and same values', () => {
 // Where the length of one list is 0
 
 
+//zip tests
+
+
+function ziplist(list1, list2) {
+  let current1 = list1.head;
+  let current2 = list2.head;
+  let newList = new LinkedList();
+  while (current1 || current2) {
+    if (current1) {
+      newList.append(current1.value);
+      current1 = current1.next;
+    }
+    if (current2) {
+      newList.append(current2.value);
+      current2 = current2.next;
+    }
+  }
+  return newList;
+}
```

Comment:
Not sure why you have this function in the test file; you want to require it from the linked list file instead.